### PR TITLE
Add recipe, patch

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
wasmtime-py 11.0.0
[upstream](https://github.com/bytecodealliance/wasmtime-py)

### CODE REVIEW

File: recipe/patches/0001-read-library-from-standard-anaconda-location.patch [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
++    libname = 'wasmtime.dll'
+ elif sys.platform == 'darwin':
+-    libname = '_libwasmtime.dylib'
++    libname = 'libwasmtime.dylib'
+ else:
+     raise RuntimeError("unsupported platform `{}` for wasmtime".format(sys.platform))
+ 
+@@ -25,10 +25,7 @@ if machine == 'arm64':
+ if machine != 'x86_64' and machine != 'aarch64':
+     raise RuntimeError("unsupported architecture for wasmtime: {}".format(machine))
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/patches/0001-read-library-from-standard-anaconda-location.patch:31]: @akabanovs that is why s390 and ppc64le are not supported

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{% set name = "wasmtime-py" %}
+{% set version = "11.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bytecodealliance/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```
Comment (akabanovs) [inline recipe/meta.yaml:9]: Could use pypi url for this. Not critical I guess.

**Review by akabanovs — APPROVED** [review]

**Review by ViridianMelody — APPROVED** [review]
LGTM! Just be sure to delete the abs.yaml and upload `wasmtime` before merge.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+
+about:
+  home: https://github.com/bytecodealliance/wasmtime-py
+  summary: python bindings for Wasmtime
+  description: |
+   The wasmtime-py package has initial support for running WebAssembly components in Python with high-level bindings. 
+
+  license_family: Apache
+  license: Apache-2.0
```
Comment (skupriienko) [inline recipe/meta.yaml:48]: `Apache-2.0 WITH LLVM-exception` https://github.com/bytecodealliance/wasmtime-py/blob/main/LICENSE#L205.
You can build and upload it again with a build number `1` to fix this license name

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:48]: yes, done and pushed to snowflake

**Pull request merged**